.github/workflows/python-tests.yml

```diff
@@ -64,9 +73,3 @@ jobs:
           else
             nox --python=${{ matrix.python }}
           fi
-      - name: Upload documentation
-        if: matrix.session == 'docs-build'
-        uses: actions/upload-artifact@v4
-        with:
-          name: docs
-          path: clients/python/docs/_build
```

Comment:
I'm not sure why this is removed, isn't this storing a preview of the docs while submitting PRs?

Reply:
afaik there's currently no way to see these. if we enable PR docs build on readthedocs it still performs the build on their side

Reply:
> afaik there's currently no way to see these

go actions
![Screenshot 2024-08-26 at 09 23 45](https://github.com/user-attachments/assets/edcd5aff-83d9-4f37-bcb4-545cd579930d)

select gha workflow for python: 
![Screenshot 2024-08-26 at 09 24 33](https://github.com/user-attachments/assets/a8cf83a0-05eb-4ed0-8300-deed32f80017)

select a run, for instance here i selected latest from main default branch
https://github.com/kubeflow/model-registry/actions/runs/10510838464

![Screenshot 2024-08-26 at 09 27 17](https://github.com/user-attachments/assets/57dec185-6b51-4e4f-b68d-277e2d8b1b0c)

you obtain a zip,
and it's the preview of the docs when this gha job is run for the PRs instead.

do we want to remove this for some reasons?

Reply:
oh I didn't know that, thanks for the pointer! I don't mind keeping it, I just thought it was kinda useless.